src/renderer/components/Editor/SquareBracketHover.tsx

```diff
@@ -0,0 +1,90 @@
+import { Box } from '@mui/material';
+
+interface BracketProps {
+  isHoveredOver: boolean;
+}
+
+interface Props extends BracketProps {
+  isLast: boolean;
+}
+
+interface HorizontalBracketProps extends BracketProps, Props {
+  top: boolean;
+}
+
+const VerticalBracketBackground = ({ isHoveredOver }: BracketProps) => {
+  const opacityValue = isHoveredOver ? 0.5 : 0;
+
+  return (
+    <Box
+      id="squareBracketBackground"
+      sx={{
+        height: 'inherit',
+        width: '8px',
+        opacity: opacityValue,
+        borderRadius: '8px',
+        backgroundColor: '#FFB355',
+
+        // positioning
+        position: 'relative',
+        left: '-5px',
+        top: '-8px',
+
+        '&:hover': {
```

Comment:
Is this CSS logic needed as well as the JS logic used above?

Reply:
Nope